src/gateio_new_coins_announcements_bot/new_listings_scraper.py

```diff
@@ -47,10 +48,19 @@ def get_announcement():
     random.shuffle(queries)
     logger.debug(f"Queries: {queries}")
     request_url = (
-        f"https://www.binancezh.com/gateway-api/v1/public/cms/article/list/query"
+        f"http://www.binance.com/gateway-api/v1/public/cms/article/list/query"
```

Comment:
Personally I think the domain shouldn't be fixed in this PR since it doesn't have anything to do with the rotating proxy feature.
I would prefer it if we fixed the domain bug in a separate PR. 
@CyberPunkMetalHead whats your opinion on this?